.buildkite/steps/test-neo4j-notebooks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -xeo pipefail
+
+echo "--- :python: installing stellargraph"
+# install stellargraph itself and the dependencies necessary for neo4j, which (hopefully) won't
+# install any other ones. (The neo4j deps aren't installed into the docker image, so the main tests
```

Comment:
I'm not sure if I understand this comment - which "other ones" are you referring to that we shouldn't be installing?

Reply:
Other dependencies, because they should be in the docker image already. I've rephrased the comment.

Reply:
Thanks, makes sense 👍